utilities/must_gather.py

```diff
@@ -16,6 +16,7 @@ def run_must_gather(
     node_name: str = "",
     flag_names: str = "",
     timeout: str = f"{TIMEOUT_15MIN}s",
+    command_timeout: str = f"{TIMEOUT_20MIN}",
```

Comment:
run_command timeout is an int, so this should also be an int

Reply:
my bad. Thanks for noticing!